miasm2/core/asmblock.py

```diff
@@ -426,11 +437,18 @@ def set_offset(self, label, offset):
         if is_int(label.offset):
             self._offset2label[label.offset] = label
 
+    @property
+    def labels(self):
+        """Return all labels"""
+        return self._labels
+
     @property
     def items(self):
+        warnings.warn('DEPRECATION WARNING: use "labels" instead of "items"')
```

Comment:
You probably want to swap this two lines, to have the docstring right after the function definition